### DISCUSSION TITLE
WebComponents V3

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2564] [General]
I saw that web-components v3 is in beta. It is completely different, the datagrid and other components are not present. What does this mean? 
Will the transition to v3 be impossible? 
Will this Blazor library always remain on v2?

Best reagds

**Comment by vnbaaij** [comment]
We are already working on the next major version of the library which will be using the v3 web components (see the dev-v5 branch and https://fluentui-blazor.azurewebsites.net/). 

Not 100% sure yet what will be available from the get go. It might be that we will slowly evolve to a more complete offering over time
We will provide comprehensive migration information but there will probably not be a full 1 on 1 conversion.

**Reply by joriverm** [reply]
while looking into an issue i saw v3 of the webcomponents had a few big refactors, and then looked into the v5 branch, i can understand now why it will take a few months. big undertaking!
